references/cli/addon.go

```diff
@@ -203,6 +205,7 @@ func NewAddonEnableCommand(c common.Args, ioStream cmdutil.IOStreams) *cobra.Com
 	cmd.Flags().StringVarP(&addonVersion, "version", "v", "", "specify the addon version to enable")
 	cmd.Flags().StringVarP(&addonClusters, types.ClustersArg, "c", "", "specify the runtime-clusters to enable")
 	cmd.Flags().BoolVarP(&skipValidate, "skip-version-validating", "s", false, "skip validating system version requirement")
+	cmd.Flags().BoolVarP(&overrideDefs, "override-defs", "o", false, "override already exist definitions within addon")
```

Comment:
```suggestion
	cmd.Flags().BoolVarP(&overrideDefs, "override", "o", false, "override existing definitions if conflict with those contained in this addon")
```

Reply:
if you're desigining other override options in the future, I would suggest the option name to be "override-definitions" and with no short args("o").